operations/operations/publishing/publishing-guidelines/voice-tone-and-writing-style-guidelines.md

```diff
@@ -1,24 +1,98 @@
 
-**Voice and Tone**
+Some of the recommendations across the documents, such as punctuating lists and heading capitalization are not currently aligned but will be iterated on for alignment across all web properties. 
+
+## Voice and Tone
+
+Our voice is informed by our foundation as a messaging platform for Enterprise Developers and DevOps, but respects that our client base spans many types of organizations. Writing is one way we can bring Mattermost’s values to life. And above all, our voice demonstrates our deep respect for our clients and users. Ultimately, Mattermost exists because of them, so it’s important we keep their needs at the forefront of our writing style.
+
+Mattermost's voice is:
+* Professional
+* Concise
+* Clear
+* Consistent
+* Customer-centric
+* Empowering
+
+To achieve this, we keep the following principles in mind when writing content: 
 
 * Use simple, plain language that is easy to understand
 * Use direct, casual tone instead of an informal tone
 * Use the active voice
+* Write clearly - avoid vague terms and jargon
+* Write short, active sentences and maintain a visual separation between page elements
 
-**Writing Style**
+The appropriate tone differs across different mediums. We don’t write help documentation with the same tone as website copy.
+You can vary your tone to fit the situation, just as you’d talk to an angry person with a different tone than with an excited child. Voice is to tone as climate is to weather. The Mattermost voice remains the same, even when the tone varies.
 
-* Write clearly and concisely - get to the point quickly and concisely without losing valuable information
-* Work on the premise that "Every page is page 1", as a large portion of users enters our documentation from a Google search
+## Writing Tenets
+
+* Work on the premise that "Every page is page 1", as a large portion of users enters our documentation from a Google search**
+* Add a summary to the top of each page for readers to be able to quickly assess the content for suitability
+* Write clearly and concisely - get to the point quickly without losing valuable information
 * Focus on what the target audience wants to accomplish by being practical/outcome-focused
 * Break long sections into smaller, easily digestable subpages
 * Have at most one key point or action per paragraph
+* Refer to one thing or idea with the same word throughout the page.
 * Use ordered lists or bullets where appropriate, as they are generally easier to read than long blocks of text
 * Write for an international audience without idioms or expressions that people outside of your region are unlikely to understand
+* Minimize content so it can be found and remembered. Keep pages short, modular, and focused on a single topic.
+
+## Writing Style
```

Comment:
Good question. This section was an attempt to distill the "ideal basics" such as numbering and I considered adding more details about markdown/rst formatting but imo that is too detailed for this guide and can be a separate "House Style" guide for use by editors and technical writers. The nitty-gritty of writing is barrier to contribution so the current doc guidelines will be a separate section in the publishing space. 

Taking this into account, it may be better to cut out this section and refer to the "house style guide" instead (when it's available) so there's less cognitive load. 

What are your thoughts?

Reply:
Hi @justinegeffen - an excellent point. I like the idea of a style guide for editors.

For this section, if there are rules or guidelines that would apply across our web properties, e.g. numbers and capitalization may be, then those are okay to keep. But more detailed guidelines around markdown/RST fit better for the docs-specific guidelines for editors.

Reply:
The two below points apply across all web properties going forward - and existing content will be updated to align with this in a phased approach (to be discussed). 

Agree that more detailed guidelines are appropriate in an editorial context but not here.

Reply:
Thanks! Agree